### PR TITLE
notOk() method based on implementation of ok()

### LINKED ISSUE #744: Add assert.notOk() function
The same way that equal(), deepEqual(), propEqual() and strictEqual() has their negative versions, ok() should have a notOk() negative version to check for falsy values, in a more clean and explicit way that by adding a negation operator in front of the value to check.

```Javascript
assert.ok(!value)
assert.notOk(value)
```

### PR DESCRIPTION
**Pull request by piranna**
This fixes #744

### CODE REVIEW

File: src/assert.js [hunk 1]
```diff
@@ -77,6 +77,18 @@ QUnit.assert = Assert.prototype = {
 	},
 
 	/**
+	 * Asserts rough false-ish result.
+	 * @name notOk
+	 * @function
+	 * @example notOk( "asdfasdf".length <= 5, "There must be at max 5 chars" );
```
Comment (leobalter) [inline src/assert.js]: This seems confusing. This assertion is not a good example.

Although, I filed the PR #746 to remove these JSDocs, let's see if we can get rid of them.

**Comment by leobalter** [comment]
Would you mind to help documenting it at [api.qunitjs.com](https://github.com/jquery/api.qunitjs.com) and signing the [jQuery CLA](http://contribute.jquery.org/CLA/)?

File: src/assert.js (hunk 1 above)
Comment (leobalter) [inline src/assert.js]: That PR is already merged, would you mind to rebase and kill the JSDoc comment?

**Comment by leobalter** [comment]
@piranna any update?

**Comment by piranna** [comment]
Sorry for the delay. I've just rebased the pull-request and also [created a new one with the documentation](https://github.com/jquery/api.qunitjs.com/pull/99).

**Comment by leobalter** [comment]
commit ready to merge at https://github.com/leobalter/qunit/tree/piranna-patch-1, I just want to check one more thing before.

**Comment by leobalter** [comment]
merged at f61926d3c45fc511842a8b04139afa044ad60636

Thanks

**Comment by piranna** [comment]
Thanks to you :-)

**Pull request closed without merging**